### ISSUE TITLE
DataView: accessibility - paginator dropdown "Form field missing a label"

### ISSUE THREAD

**Issue by markuczy** [primefaces/primeng#14891]
### Describe the bug

Hi, I am working on a component which uses DataView component. Accessibility checker tools are reporting an issue with an input element, which is a child of p-dropdown (p-dataView -> p-paginator -> p-dropdown).

The following line is what is causing the issue: `<input type="text" readonly="" aria-haspopup="listbox" pautofocus="" role="combobox" class="p-element" aria-expanded="false">`.  The violated rule is this - https://dequeuniversity.com/rules/axe/4.8/label.

### Environment

Ubuntu 22.04, WSL 2, Windows 11 PRO

### Reproducer

https://stackblitz.com/edit/github-j1atbu

### Angular version

15.2.10

### PrimeNG version

15.2.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

### Browser(s)

Chrome 121

### Steps to reproduce the behavior

1. Use dataView component with paginator and rowPerPageOptions
2. Validate accessibility with lighthouse/siteImprove/axa
3. Expect "Form field missing a label" 

### Expected behavior

When using dataView with paginator and rowsPerPageOptions accessibility tests are not reporting any issues.
